### PR TITLE
🔖💀 0.9.0-gonk

### PR DESCRIPTION
**Pull request by effs**
REDmod autoconversion enabled by default

### CODE REVIEW

File: src/features.ts [hunk 1]
```diff
@@ -9,5 +9,9 @@ export interface Features {
 }
 
 export const CurrentFeatureSet: Features = {
+  REDmodAutoconvertArchives: Feature.Enabled,
+};
+
+export const DefaultFeatureSetForTesting: Features = {
   REDmodAutoconvertArchives: Feature.Disabled,
 };
```
Comment (E1337Kat) [inline src/features.ts:17]: These should be congruous. The production features set and testing feature set shouldn't be different, imo. and just set the feature as disabled for the old style archive install

**Review by E1337Kat — CHANGES_REQUESTED** [review]
I don't think we should have the feature be enabled by default without a settings UI to disable it or not.

File: src/features.ts (hunk 1 above)
Comment (effs) [reply src/features.ts:17]: > I don't think we should have the feature be enabled by default without a settings UI to disable it or not.

It's the only way to test it for now.

I'm not sure what the best approach is (especially if the features are togglable, which means there is no prod features set as such), but there has to be a baseline for tests, or then every test has to be updated to specify the feature set it uses. Or if we switch the baseline to enabled conversion, then there's ~25 tests that need to be updated.

Comment (E1337Kat) [reply src/features.ts:17]: hmm.... ok. I guess if this isn't the nexus upload it is okay... fair point on the tests though.

**Review by E1337Kat — APPROVED** [review]

**Comment by effs** [comment]
Merged already

**Pull request closed without merging**